dxr/plugins/clang/condense.py

```diff
@@ -248,11 +254,33 @@ def condense_file(csv_folder, file_path):
         compiler plugin
     :arg file_path: A path to the file to analyze, relative to the tree's
         source folder
+    :arg overrides: A dict whose keys are function qualnames that are overrides
+    :arg overriddens: A dict whose keys are function qualnames that are
+        overriddens
 
     """
+    def process_function_for_override(overrides, overriddens, props):
```

Comment:
Is there any reason for these to be inner functions? Most of the rest aren't, and I loathe inconsistency. :-) I see 2 alternatives:
 1. Make the functions module-level, and use partials.
 2. Keep the functions local, but rely on closing over `overrides` and `overriddens` rather than passing them in, so we have an excuse to keep them local.

In any case, `dispatch_table` would remain local, as you've made it.

I don't think there's a need to both make `dispatch_table` *and* the functions local.